bitnami/elasticsearch/8/debian-12/rootfs/opt/bitnami/scripts/libelasticsearch.sh

```diff
@@ -909,6 +909,9 @@ elasticsearch_healthcheck() {
     local host
 
     host=$(get_elasticsearch_hostname)
+    if validate_ipv6 "$host"; then
+        host="[$host]"
```

Comment:
Same here

```suggestion
        host="[${host}]"
```

Reply:
Should make no difference in case of a validated IPv6 address, but I added the braces. I added them in the previous
line for consistency as well.

Reply:
Thanks